test/parallel/test-http-client-keep-alive-release-before-finish.js

```diff
@@ -4,16 +4,18 @@ const http = require('http');
 
 const server = http.createServer((req, res) => {
   res.end();
-}).listen(common.PORT, common.mustCall(() => {
+}).listen(0, common.mustCall(() => {
   const agent = new http.Agent({
     maxSockets: 1,
     keepAlive: true
   });
 
+  const port = server.address().port;
+
   const post = http.request({
     agent: agent,
     method: 'POST',
-    port: common.PORT,
+    port: port,
```

Comment:
Same could be said for `agent` as well.

    const post = http.request({
      agent,
      port,
      method: 'POST'
    });